smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -1133,7 +1133,7 @@ def setColor(childDevice, huesettings) {
 			value.hue = Math.min(Math.round(huesettings.hue * 65535 / 100), 65535)
 		if (huesettings.saturation != null)
 			value.sat = Math.min(Math.round(huesettings.saturation * 254 / 100), 254)
-	} else if (huesettings.hex != null && false) {
```

Comment:
So now else if block could get executed where as before it wouldn't. Is that what you intended?